pkgs/by-name/an/anytype/package.nix

```
@@ -1,40 +1,118 @@
-{ lib, fetchurl, appimageTools, makeWrapper, commandLineArgs ? "" }:
+{
+  lib,
+  callPackage,
+  fetchFromGitHub,
+  buildNpmPackage,
+  pkg-config,
+  libsecret,
+  electron,
+  makeDesktopItem,
+  copyDesktopItems,
+  commandLineArgs ? "",
+}:
 
 let
+  anytype-heart = callPackage ./anytype-heart { };
   pname = "anytype";
-  version = "0.43.8";
-  name = "Anytype-${version}";
-  src = fetchurl {
-    url = "https://github.com/anyproto/anytype-ts/releases/download/v${version}/${name}.AppImage";
-    hash = "sha256-inqJvx5K/k97X50E0FYlzJDKqrVjAU6ZKIVdCWHr8NI=";
+  version = "0.44.0";
+  src = fetchFromGitHub {
+    owner = "anyproto";
+    repo = "anytype-ts";
+    rev = "v${version}";
+    sha256 = "sha256-a2ZnTEAFzzTb+lxtQkC6QLG5SP1+gDSjI9dqUNZWfCg=";
```

Can we use `tag` and `hash` here since we are using the tag version releases?
```suggestion
    tag = "v${version}";
    hash = "sha256-a2ZnTEAFzzTb+lxtQkC6QLG5SP1+gDSjI9dqUNZWfCg=";
```